### DISCUSSION TITLE
Multi-Zone Controller, manual start multiple zones

### DISCUSSION THREAD

**Discussion by flacco54** [sebr/bhyve-home-assistant#163] [Ideas]
Hi,

Loving the integration and have been using it for over a year now. I had an issue and didn't realise I was bound by the 10min default run-time. I have a multi-zone controller and I cannot update this default run-time. So I updated my automation to call the bhyve.start_watering service while also declaring the minutes.

All is well, but I end up with a heap of code as I have 6 zones to control. By that, I mean I need to add separate calls for each zone with a delay between each. Now, in the app, if I manually water I can 'queue' up say 12 minutes for zone 1, 16 minutes for zone 2 and on and on it goes. Then the program starts.

Is it at all possible to send a bhyve api call to queue up multiple zones on a single multi-zone controller? It would make it so much better for automations.

**Comment by sebr** [comment]
> Loving the integration and have been using it for over a year now

Thank you <3

> I have a multi-zone controller and I cannot update this default run-time.

I have heard of others also having this problem. Were you able to change it in the B-Hyve app?

> Is it at all possible to send a bhyve api call to queue up multiple zones on a single multi-zone controller? It would make it so much better for automations.

Unfortunately not that I've discovered. It doesn't mean it's not possible... and in theory could be done through some custom code in the integration however tbh I don't have plans to look into this in the short term.

There is an existing issue which you can subscribe to for future updates or community contributions: https://github.com/sebr/bhyve-home-assistant/issues/43
